manuscript.md

```diff
@@ -12,6 +16,30 @@
 
 ## What is currently limiting our ability to predict network structure?
 
+* As this will probably play a part in up setting the tone/concept for the ms 
+I guess it makes sense to make sure we address the need for ‘tools’ (as 
+granted by the all seeing light of ML) That being said might be nice to bring 
+in a general point for each section - if at all possible...
+
+The scarcity of data on species interactions is certainly one of the biggest 
```

Comment:
Needs a sentence explaining where the scarcity is from: (i) detecting species, (ii) detecting species together, (iii) detecting species interactions